actions/watch.ts

```diff
@@ -320,9 +374,18 @@ async function placeOrder(orderUid: string, order: any, apiUrl: string) {
     const errorMessage = "[placeOrder] Error placing order in API";
     if (error.response) {
       const { status, data } = error.response;
+
+      const { shouldThow } = handleOrderBookError(status, data);
```

Comment:
For some error we will want to throw, and for some others, we will want to ignore. `handleOrderBookError` will tell us what we should do